client-app/ui-kit/composables/useQuantityValidationSchema.test.ts

```diff
@@ -0,0 +1,121 @@
+import { describe, expect, it } from "vitest";
+import { mockI18n } from "../mocks";
+import { useQuantityValidationSchema } from ".";
+
+describe("use-quantity-validation-schema", () => {
+  mockI18n();
+
+  it("quantity is positive", () => {
+    const { quantitySchema } = useQuantityValidationSchema({});
+
+    expect(quantitySchema.value.isValidSync(1)).toBeTruthy();
+    expect(quantitySchema.value.isValidSync(0)).toBeFalsy();
+    expect(quantitySchema.value.isValidSync(-1)).toBeFalsy();
+  });
+
+  it("quantity is integer", () => {
+    const { quantitySchema } = useQuantityValidationSchema({});
+
+    expect(quantitySchema.value.isValidSync(1)).toBeTruthy();
+    expect(quantitySchema.value.isValidSync(1.5)).toBeFalsy();
+  });
+
+  it("quantity is less than system limit", () => {
+    const { quantitySchema } = useQuantityValidationSchema({});
+
+    expect(quantitySchema.value.isValidSync(1)).toBeTruthy();
+    expect(quantitySchema.value.isValidSync(999999999)).toBeFalsy();
```

Comment:
I'd like better to use LINE_ITEM_QUANTITY_LIMIT + 1 here for clarity, because with named eye it's hard to say is there a correct amount of 9's :)